### PR TITLE
Feat boxplot

### LINKED ISSUE #12: Update boxplot function

### PR DESCRIPTION
**Pull request by miyer26**

### CODE REVIEW

File: R/boxplot.R [hunk 1]
```diff
@@ -6,9 +6,93 @@
...
+#check if y exists in df
+if ( !(as.character(as.list(
+        rlang::get_expr(ggplot2::vars({{ y }})[[1]])
+    )))
+    %in%
+    colnames({{ df }})
+){
+    stop('y column name not found')
+}
+
```
Comment (iamMoid) [inline R/boxplot.R:60]: You should also add tests to check if the data type of x is a "character"/"factor" and that of y is "numeric"/"float"/"int" in the dataframe.

File: R/boxplot.R [hunk 2]
```diff
@@ -6,9 +6,93 @@
...
+
+
+
+x_title <- str_replace(as.character
```
Comment (iamMoid) [inline R/boxplot.R]:
```suggestion
x_title <- stringr::str_replace(as.character
```

File: R/boxplot.R [hunk 3]
```diff
@@ -6,9 +6,93 @@
...
+        if (facet == TRUE){
+            return(plot  +
+                ggplot2::facet_wrap(ggplot2::vars({{ x }}))
+            )
+        } else {
+            return(plot)
+        }
+return(plot)
 }
+
```
Comment (iamMoid) [inline R/boxplot.R]: Can you look into indexing as well? Not a top priority for now but as and when time permits.

File: R/boxplot.R [hunk 4]
```diff
@@ -6,9 +6,93 @@
 #' @param facet Determines whether separate graphs will be created for each category
 #'
 #' @return a ggplot object
+#'
+#'
+#'
```
Comment (iamMoid) [inline R/boxplot.R]:
```suggestion
```

File: R/boxplot.R [hunk 5]
```diff
@@ -6,9 +6,93 @@
...
 #' @export
 #'
 #' @examples
 #' function(df, Length, Species, TRUE)
```
Comment (iamMoid) [inline R/boxplot.R]:
```suggestion
#' > boxplot(df, Length, Species, TRUE)
```

**Review by iamMoid — CHANGES_REQUESTED** [review]
Good work! Made a few comments. Should be good to go once addressed.

File: R/boxplot.R (hunk 2 above)
Comment (miyer26) [reply R/boxplot.R]: Addressed

File: R/boxplot.R (hunk 4 above)
Comment (miyer26) [reply R/boxplot.R]: Addressed

File: R/boxplot.R (hunk 5 above)
Comment (miyer26) [reply R/boxplot.R]: Whoops, good spot

File: R/boxplot.R (hunk 1 above)
Comment (miyer26) [reply R/boxplot.R:60]: Good suggestion!

**Review by iamMoid — APPROVED** [review]

**Comment by iamMoid** [comment]
Looks good now!

**Pull request merged**